Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [scripts]
-test = "pytest ./tests --cov=pythoncz --pylama"
+test = "python -m pytest ./tests --cov=pythoncz --pylama"
```

Comment:
This adds current directory to the Python path and makes imports working

Reply:
Seems strange. So pipenv install does not install pythoncz?

Reply:
There is no more pythoncz to install.

Reply:
Also, documented at https://docs.pytest.org/en/latest/usage.html#calling-pytest-through-python-m-pytest